app/bundles/CoreBundle/Controller/BuilderControllerTrait.php

```diff
@@ -31,7 +31,7 @@ protected function getAssetsForBuilder()
             ->addScriptDeclaration("var mauticAjaxUrl     = '".$routerHelper->generate('mautic_core_ajax')."';")
             ->addScriptDeclaration("var mauticBaseUrl     = '".$routerHelper->generate('mautic_base_index')."';")
             ->addScriptDeclaration("var mauticAssetPrefix = '".$assetsHelper->getAssetPrefix(true)."';")
-            ->addScriptDeclaration("var mauticLang         = '".$this->get('templating.helper.translator')->getJsLang()."';")
+            ->addScriptDeclaration("var mauticLang        = '".str_replace("'", "\\'", $this->get('templating.helper.translator')->getJsLang())."';")
```

Comment:
Hello @alanhartless,,

The $view is only accessible in template files. I have not found an equivalent for Trait classes. Can you tell me if I'm missing something?

Thanks